store/src/main/java/org/apache/rocketmq/store/MappedFileQueue.java

```diff
@@ -39,11 +39,15 @@
 
     private final int mappedFileSize;
 
-    private final CopyOnWriteArrayList<MappedFile> mappedFiles = new CopyOnWriteArrayList<MappedFile>();
+    private final CopyOnWriteArrayList<MappedFile> mappedFiles = new CopyOnWriteArrayList<>();
 
     private final AllocateMappedFileService allocateMappedFileService;
 
-    private long flushedWhere = 0;
```

Comment:
Consider the abuse of `offset` in rocketmq-store, so we use `flushedWhere` and `committedWhere`.

`flushedWhere` means the offset of all mappedFile, while `offset` means the offset of one mappedFile in general.